bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiPoePortThingHandler.java

```diff
@@ -132,6 +135,17 @@ protected State getChannelState(final Map<Integer, UniFiPortTuple> ports, final
         return state;
     }
 
+    private State setOfflineOnNoPoEPortData() {
+        if (getThing().getStatus() != ThingStatus.OFFLINE) {
+            logger.debug(
+                    "No PoE port information for thing '{}' could be found in the UniFi API data for port '{}'. Setting thing off line.",
+                    getThing().getUID(), config.getPortNumber());
```

Comment:
This seems unneeded as framework will already log the status change. Instead, consider adding port number to the description if this is useful.